docs/CONFIG-PROPERTIES.md

```diff
@@ -4,13 +4,16 @@
 | ---- | ---- | ------- | ----------- |
 | app.allow.vnc | boolean | false (only local access) | allow access to EVE's VNC ports from external IPs |
 | app.fml.resolution | string | notset | Set system-wide value of forced resolution for applications running in FML mode, it can be one of [predefined](/pkg/pillar/types/global.go) FmlResolution* values. |
-| timer.config.interval | integer in seconds | 60 | how frequently device gets config |
+| timer.config.interval | integer in seconds | 60 | how frequently device gets config (needs a reboot to take effect) |
 | timer.cert.interval | integer in seconds | 1 day (24*3600) | how frequently device checks for new controller certificates |
 | timer.metric.interval | integer in seconds | 60 | how frequently device reports metrics |
+| timer.hardwarehealth.interval | integer in seconds | 12 hours (43200) | how frequently device reports hardware health information (ECC) to controller |
```

Comment:
The naming is unclear here...
Why is "hardwarehealth" ECC and "hardwareinfo" SMART?

Reply:
We introduce "hardwarehealth" in this PR... Could we choose a better name... Now these two are really confusing.

Reply:
nono, `hardwarehealth` was introduced in https://github.com/lf-edge/eve/pull/4677

Reply:
In #4677 it's explained on both PR description and in the commit message that for now it reports only ECC but more resources are expected to be added in the future. I would change a bit the phrase here @europaul to something like this:

 `...health information (currently only memory ECC) to controller`

Regarding documentation. Indeed, original PR missed the doc. update.